src/qibocal/protocols/characterization/flux_dependence/qubit_crosstalk.py

```diff
@@ -101,24 +101,53 @@ def _acquisition(
         pulses=[qd_pulses[qubit] for qubit in qubits],
         type=SweeperType.OFFSET,
     )
-
-    delta_bias_range = np.arange(
-        -params.bias_width / 2, params.bias_width / 2, params.bias_step
-    )
     # TODO : abstract common lines with qubit flux dep routine
     if params.flux_qubits is None:
         flux_qubits = list(platform.qubits.keys())
     else:
         flux_qubits = params.flux_qubits
-    bias_sweepers = [
-        Sweeper(
-            Parameter.bias,
-            delta_bias_range,
-            qubits=[platform.qubits[flux_qubit]],
-            type=SweeperType.OFFSET,
+    if params.flux_pulses:
+        qf_pulses = {}
+        for i, qubit in enumerate(qubits):
+            if isinstance(params.flux_amplitude_start, list):
+                flux_amplitude_start = params.flux_amplitude_start[i]
+                flux_amplitude_end = params.flux_amplitude_end[i]
+                flux_amplitude_step = params.flux_amplitude_step[i]
+            else:
+                flux_amplitude_start = params.flux_amplitude_start
+                flux_amplitude_end = params.flux_amplitude_end
+                flux_amplitude_step = params.flux_amplitude_step
+            delta_bias_flux_range = np.arange(
+                flux_amplitude_start,
+                flux_amplitude_end,
+                flux_amplitude_step,
+            )
+            pulse = platform.create_qubit_flux_pulse(
+                qubit, start=0, duration=sequence.duration
+            )
```

Comment:
I would like to test if we get nice results regarding the flux pulse starting at the same time than the sequence rather than delaying the sequence a bit for the the qubit frequency to shift properly. I would assume this would not be relevant for resonator experiments but could be for qubit ones. (Ignore the -100, but I assume that could be a nice value to delay the sequence after the flux pulse start.)

```suggestion
            pulse = platform.create_qubit_flux_pulse(
                qubit, start=-100, duration=sequence.duration
            )
```